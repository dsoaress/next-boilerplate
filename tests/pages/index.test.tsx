import { render } from '@/lib/testUtils'
import IndexPage from '@/pages/index'

describe('<IndexPage />', () => {
  it('Renders IndexPage', () => {
    const { container } = render(<IndexPage />)

    expect(container).toMatchSnapshot()
  })
})
