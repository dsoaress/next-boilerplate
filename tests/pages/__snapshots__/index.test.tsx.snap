// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IndexPage /> Renders IndexPage 1`] = `
.c0 {
  max-width: 1280px;
  padding: 0 0.5rem;
  margin: 0 auto;
}

<div>
  <main
    class="c0"
  >
    <h1>
      Hello
    </h1>
  </main>
</div>
`;
